docker-compose.yml

```
@@ -0,0 +1,11 @@
+services:
+  postgres:
+    image: postgres:14-alpine
+    ports:
+      - 9667:5432
+    volumes:
+      - ./apps/postgres:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_PASSWORD=65465
+      - POSTGRES_USER=mc
+      - POSTGRES_DB=stationofdoom
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)